File: src/main/java/com/example/electronicdigitalsignature/data/PrivateKey.java
package com.example.electronicdigitalsignature.data;

import java.math.BigInteger;

public class PrivateKey {
    private BigInteger w;

    public PrivateKey(BigInteger w) {
        this.w = w;
    }

    public BigInteger getW() {
        return w;
    }

    public void setW(BigInteger w) {
        this.w = w;
    }

    @Override
    public String toString() {
        final StringBuilder sb = new StringBuilder("PrivateKey{");
        sb.append("w=").append(w);
        sb.append('}');
        return sb.toString();
    }
}

File: src/main/kotlin/xyz/le30r/data/PrivateKey.kt
package xyz.le30r.data

import java.math.BigInteger

class PrivateKey(var w: BigInteger) {

    override fun toString(): String {
        val sb = StringBuilder("PrivateKey{")
        sb.append("w=").append(w)
        sb.append('}')
        return sb.toString()
    }
}